<launch>
	<arg name="teleop_args" default="-rc" />

	<node pkg="remote_sensing_mapping_uav" type="teleop.py" name="teleop" args="$(arg teleop_args)" required="True" output="screen">
	</node>
</launch>